### PR TITLE
Changelog release 0.5.3

### PR DESCRIPTION
**Pull request by heinezen**
Bugfix and refactoring release for `0.5.2`.

- [x] Changelog
    - [x] Release date
    - [x] Version number in title
    - [x] Full commit log
- [x] Bump version in `openage_version`

### CODE REVIEW

File: doc/changelogs/engine/v0.5.3.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) since release [0.4.0].
+
+## Added
+
+- Temporary file/dorectory support for Python files
+- More debug info in converter
+- More fixed-point math functions
+- `setuptools` is now conditional dependemcy for Python >= 3.12 && Cython < 3.1
```
Comment (TheJJ) [inline doc/changelogs/engine/v0.5.3.md:12]: demendemcy

File: doc/changelogs/engine/v0.5.3.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# [0.5.3] - 2023-12-15
+All notable changes for version [0.5.3] are documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) since release [0.4.0].
+
+## Added
+
+- Temporary file/dorectory support for Python files
```
Comment (TheJJ) [inline doc/changelogs/engine/v0.5.3.md:9]: dorectory

File: doc/changelogs/engine/v0.5.3.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - Logic (`libopenage/presenter/legacy`)
+    - Data classes (texure, etc.)
+  - Old Terrain (`libopenage/terrain`)
+
+## Fixed
+
+- Version tag format without `--long` crashes on tagged commits
+- Dangling reference in modpack info file loading
+- No graphics visible on Wayland
+- Wtonh anchor positions when sprite is mirrored
```
Comment (TheJJ) [inline doc/changelogs/engine/v0.5.3.md:44]: wtonh?

Comment (heinezen) [reply doc/changelogs/engine/v0.5.3.md:44]: Wrong :P

File: doc/changelogs/engine/v0.5.3.md (hunk 1 above)
Comment (heinezen) [reply doc/changelogs/engine/v0.5.3.md]: fixed

File: doc/changelogs/engine/v0.5.3.md (hunk 2 above)
Comment (heinezen) [reply doc/changelogs/engine/v0.5.3.md]: fixed

**Comment by heinezen** [comment]
All fixed and ready

**Review by TheJJ — APPROVED** [review]

**Pull request merged**